src/components/Button.tsx

```diff
@@ -6,14 +6,18 @@ type ButtonProps = {
   buttonText: string;
   onClick?: MouseEventHandler<HTMLButtonElement>;
   className?: string;
+  buttonIcon?: any;
 }
 
 const Button = ({
-  buttonType, buttonText, onClick, className
+  buttonType, buttonText, onClick, className, buttonIcon
 }: ButtonProps) => {
   return (
     <Fragment>
-      <button type={buttonType} className={className} onClick={onClick}>{buttonText}</button>
+      <button type={buttonType} className={className} onClick={onClick}>
+        {<>{buttonIcon}</>}
```

Comment:
Do we really need React fragment <> here:

{<>{buttonIcon}</>} ?

Simply {buttonIcon} doesn`t work?

Reply:
:) It's interesting fact, earlier I tried to use {buttonIcon}, and it doesn't work, but now it work. Done